.eslint-dictionary.json

```diff
@@ -388,5 +401,6 @@
   "xcode",
   "xcodeproj",
   "yaml",
-  "yyyymmddhhmmss"
-]
\ No newline at end of file
+  "yyyymmddhhmmss",
+  "zoneinfo"
```

Comment:
nit: can this be replaces in code to zoneInfo

Reply:
This one looks to be a string literal used in an external service.